DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/n_way_buffering/src/n_way_buffering.cpp

```diff
@@ -210,11 +215,11 @@ int main() {
 #endif
 
   try {
-    auto prop_list =
-        property_list{property::queue::enable_profiling()};
+    auto prop_list = property_list{property::queue::enable_profiling()};
 
     std::unique_ptr<queue> q;
-    q.reset(new queue(device_selector, dpc_common::exception_handler, prop_list));
```

Comment:
Just looking at this code randomly (well, attracted by the word FPGA :-) )... What are these `std::unique_ptr<queue> q` everywhere?
In SYCL a `queue` is already a kind of `std::shared_ptr` so this seems uselessly dangerous to have .

Reply:
Agree, this is dumb and misleading. It used to be widespread in the FPGA samples; I thought they'd all been mopped up.

This particular change isn't introducing these -- it got highlighted by clang-format. I'll file a case to the sample owner to clean up this mess, but it will be on a separate review.